client/consensus/src/lib.rs

```diff
@@ -149,6 +154,22 @@ impl<B, I, C> BlockImport<B> for FrontierBlockImport<B, I, C> where
 					}
 				},
 			}
+			// On importing block 1 we also map the genesis block in the auxiliary.
+			if block.header.number().clone() == One::one() {
+				let id = BlockId::Number(Zero::zero());
+				if let Ok(Some(header)) = client.header(id) {
+					let block = self.client.runtime_api().current_block(&id)
```

Comment:
I'm not too worried about this because it only happens once.

But out of curiosity, will we be able to avoid the runtime api once #199 lands?

Reply:
Yes, we can use the `StorageProvider` from the import pipeline.